contracts/FlowIDTableStaking.cdc

```diff
@@ -38,6 +38,7 @@ access(all) contract FlowIDTableStaking {
     /// Epoch
     access(all) event NewEpoch(totalStaked: UFix64, totalRewardPayout: UFix64, newEpochCounter: UInt64)
     access(all) event EpochTotalRewardsPaid(total: UFix64, fromFees: UFix64, minted: UFix64, feesBurned: UFix64, epochCounterForRewards: UInt64)
+    access(all) event NodeRewardsSetToSlash(nodeSlashPercentagesRemaining: {String: UFix64})
```

Comment:
I think it's useful to differentiate between withholding rewards and slashing. In my view, slashing is a punitive measure for a protocol violation where part of a node's initial stake amount is confiscated. 

I don't think we should refer to withholding rewards as slashing in general. In particular if we plan to pursue performance-based rewards, that blurs the line even further between "getting more rewards for good performance" and "getting less rewards for worse performance".

```suggestion
    // The single parameter is a map from node ID to the percentage of rewards that will be withheld from
    // node at each subsequent payout, until the withholding map is modified again.
    access(all) event NodeRewardWithholdingsUpdated(nodeRewardWithholdings: {String: UFix64})
```

Reply:
yes, you're right, I forgot about the naming distinction. Thanks for the correction! I'll update it